google-cloud-spanner/src/test/java/com/google/cloud/spanner/ReadAsyncTest.java

```diff
@@ -185,6 +185,10 @@ public void pointReadNotFound() throws Exception {
 
   @Test
   public void invalidDatabase() throws Exception {
+    if (isMultiplexedSessionsEnabled()) {
```

Comment:
Same here: I don't think you need the if statement.

Reply:
Removed.